packages/components/src/filters/index.js

```diff
@@ -224,15 +231,7 @@ ReportFilters.defaultProps = {
 	query: {},
 	showDatePicker: true,
 	onDateSelect: () => {},
-	currency: new Currency( {
-		code: 'USD',
-		precision: 2,
-		symbol: '$',
-		symbolPosition: 'left',
-		decimalSeparator: '.',
-		thousandSeparator: ',',
-		priceFormat: '%1$s%2$s',
-	} ),
+	currency: Currency().getCurrency(),
```

Comment:
Not introduced by this PR, so I'm not requesting a change in this PR, but I think the naming of the `Currency()` and `getCurrency()` are confusing 🤯, especially since there are also functions such as `formatCurrency()`, which doesn't take a 'currency', but rather an amount (a number)...

Ideally, things would be named something like...

`Currency()` -> `CurrencyFactory()`
`getCurrency() -> `getCurrencyConfig()`
`formatCurrency() -> `formatAmount()`

Reply:
Absolutely, thanks for surfacing this! Here is an issue to track and tackle in the next cooldown period: https://github.com/woocommerce/woocommerce-admin/issues/4394

Reply:
Thanks for creating the new issue!